src/sage/graphs/generators/families.py

```diff
@@ -4430,3 +4430,318 @@ def CubeConnectedCycle(d):
         G.add_edge((x, y), (x ^ (1 << y), y))
 
     return G
+
+
+def StaircaseGraph(n):
+    r"""
+    Return a staircase graph with `2n` nodes
+
+    For `n \geq 3`, the staircase graph of order `2n` is the graph obtained
+    from the ladder graph of order `2n - 2`, i.e., ``graphs.LadderGraph(n - 1)``
+    by introducing two new nodes `2n - 2` and `2n - 1`, and then joining the
+    node `2n - 2` with `0` and `n - 1`, the node `2n - 1` with `n - 2` and
+    `2n - 3`, and the nodes `2n - 2` and `2n - 1` with each other.
+
+    Note that ``graphs.StaircaseGraph(4)`` is also known as the `Bicorn`
```

Comment:
the terms `bicorn` and `graph` must have double ``` ``bicorn`` ```

Reply:
Should not it be just italic as it is the definition?
(Yeah, I just realised that in some instances where I actually meant italic, it did: \`text\` instead of \*text\*).

Reply:
I have made it:
the ` ``Bicorn graph`` `.